app/uk/gov/hmrc/agentclientrelationships/services/AgentUserService.scala

```diff
@@ -36,12 +36,13 @@ class AgentUserService @Inject()(
   def getAgentUserFor(
```

Comment:
and the method name could be `getAgentAdminUserFor `

Reply:
👍